data/ecosystems/c/data/ecosystems/m/cellana-finance.toml

```diff
@@ -0,0 +1,6 @@
+# Ecosystem Level Information
+title = "Cellana Finance"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/CellanaFinance/core"]
```

Comment:
This is not an organization link. Could you update this to be the org link like `https://github.com/CellanaFinance`

Reply:
we updated